src/main/scala/symsim/concrete/ConcreteExactRL.scala

```diff
@@ -33,3 +33,24 @@ trait ConcreteExactRL[State, ObservableState, Action]
   /** Convert the matrix Q after training into a Policy map. */
   def qToPolicy (q: Q) (using Ordering[Double]): Policy =
     q.states.map { s => (s, vf.bestAction (q) (s)) }.to (Map)
+
+  def policyEval (policies: List[Policy]): List[List[Double]] =
+    val initials = Randomized.repeat (agent.initialize).take (5)
+    val rL = for
+      s_t <- initials
+    yield policies.map (policy => runEval (policy, s_t, 0.0))
+    rL.toList
+
+  def runEval(policy: Policy, s_t: State, r_acc: Double): Double =
```

Comment:
could we use `s_0` or `initial` instad of `s_t` (the `t` seems to have no meaning here.

Also why can't we just remove `r_acc`? This is not being run accumulatively, each `runEval` is separate, isn't it?